MapboxNavigation/MGLMapView.swift

```diff
@@ -121,6 +118,11 @@ extension MGLMapView {
     }
 }
 
+@objc
 protocol NavigationMapViewDelegate: class  {
```

Comment:
This protocol should probably go in NavigationMapView.swift alongside NavigationMapView.